pkg/peertls/extensions/extensions_test.go

```diff
@@ -0,0 +1,245 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package extensions_test
+
+import (
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testpeertls"
+	"storj.io/storj/pkg/identity"
+	"storj.io/storj/pkg/peertls/extensions"
+	"storj.io/storj/pkg/peertls/tlsopts"
+)
+
+func TestParseExtensions(t *testing.T) {
```

Comment:
The file that originally contained this test was moved... this test is bad but does cover some things and needs to be split up and cleaned up. While I did originally write it, it was a while ago - don't judge me. I made the minimal change to keep it working for now.

Reply:
lol